Addition of AI guidelines in the readme (#1050)
NeTEx-CEN/NeTEx: README.md

_Upcoming: templates for Pull Request_

### The use of AI in this repo 🤖

> In this repository, the use of AI should follow these guidelines. The guiding principle is that any contributor is ultimatly reponsible for their work on NeTEx and they should be the last reviewer of any AI work before submitting it for peer-review; in other words, only contributors can be authors or co-authors of any change made to NeTEx.

Reviewer: ```suggestion
In this repository, the use of AI should follow these guidelines. The guiding principle is that any contributor is ultimately reponsible for their work on NeTEx and they should be the last reviewer of any AI-guided work before submitting it for peer-review; in other words, only contributors can be authors or co-authors of any change made to NeTEx.
```

> In this repository, the use of AI should follow these guidelines. The guiding principle is that any contributor is the ultimate reponsible for their work on NeTEx and they should be the last reviewer of any AI work before submitting it for peer-review; in other words, only contributors can be autors or co-authors of any change made to NeTEx.

Reviewer: ```suggestion
In this repository, the use of AI should follow these guidelines. The guiding principle is that any contributor is ultimately reponsible for their work on NeTEx and they should be the last reviewer of any AI work before submitting it for peer-review; in other words, only contributors can be authors or co-authors of any change made to NeTEx.
```

AI **can** be used for:
- Generation of code for contributors to work on the XSD by themselves,
- Analysis of any change proposed to the XSD before it is edited / commented / approved,

> - Improve the general language in the documentation (i.e, anything in `<xsd:annotation>` and `<xsd:documentation>`) or to check for typos in any work.

Reviewer: ```suggestion
- Improvements to the general language in the documentation (i.e, anything in `<xsd:annotation>` and `<xsd:documentation>`) or to check for typos in any work.
```

> - Improve the general language in the documentation (i.e, anyting in `<xsd:annotation>` and `<xsd:documentation>`) or to check for typos in any work.

Reviewer 2: ```suggestion
- Improve the general language in the documentation (i.e, anything in `<xsd:annotation>` and `<xsd:documentation>`) or to check for typos in any work.
```

AI **cannot** be used for:
- Committing a change in a Pull Request, i.e. the AI agent cannot be an author or co-author of any commit,
- Authoring a change, i.e. the AI agent cannot be an author or co-author of any Pull Request,
- Automated code review as a stand alone, i.e. the result of a review must be proof-read by a contributor,

> - Produce documentation from scratch (i.e, anyting in `<xsd:annotation>` and `<xsd:documentation>` should be proof-read by a contributor if it was suggested by AI).

Reviewer 2: ```suggestion
- Produce documentation from scratch (i.e, anything in `<xsd:annotation>` and `<xsd:documentation>` should be proof-read by a contributor if it was suggested by AI).
```
